server.js

```diff
@@ -88,6 +88,7 @@ function handleFailure(req, res) {
    return function (err) {
       var message = less.formatError(err, {color:false})
       l("Got Error: " + req.url + "\n" +  message)
+      l(err);
```

Comment:
I would suggest making [console.trace](https://www.google.com/search?q=console.trace&rlz=1C5CHFA_enUS700US702&oq=console.trace&aqs=chrome..69i57j0l5.925j0j1&sourceid=chrome&ie=UTF-8) This way you always get the trace